Snowdog_Acm/templates/content/view/sliders/list.phtml

```diff
@@ -140,7 +140,7 @@ $sliderItemClass = 'slider__item';
             ]);
             ?>
         <div class="<?= $block->escapeHtmlAttr($sliderItemClass) ?>">
-            <<?= $bannerUrl ? 'a href="' . $bannerUrl . '" target="' . $bannerUrlTarget . '"' : 'div'; ?>
+            <<?= $bannerUrl ? 'a href="' . $block->escapeHtmlAttr($bannerUrl) . '" target="' . $block->escapeHtmlAttr($bannerUrlTarget) . '"' : 'div'; ?>
```

Comment:
second attribut `target` should have value as `/* @noEscape */` as it returns string _blank or _self